src/state/opamSysInteract.ml

```diff
@@ -774,12 +774,13 @@ let packages_status ?(env=OpamVariable.Map.empty) config packages =
     let get_installed str_pkgs =
       (* ouput:
          >ii  uim-gtk3                 1:1.8.8-6.1  amd64    Universal ...
-         >ii  uim-gtk3-immodule:amd64  1:1.8.8-6.1  amd64    Universal ...
+         >ri  uim-gtk3-immodule:amd64  1:1.8.8-6.1  amd64    Universal ...
```

Comment:
Worth expanding this a bit (looking at the `dpkg-query` man page) that the first code can be `[uihrp]` but that we only care about the second character?

Reply:
New comment added below looks perfect, thanks!